mitosheet/src/components/toolbar/ToolbarCodeDropdown.tsx

```diff
@@ -39,6 +39,7 @@ const ToolbarCodeDropdown = (props: ToolbarCodeDropdownProps): JSX.Element => {
                     : <></>
                 }
                 {makeToolbarDropdownItem(props.actions[ActionEnum.CODESNIPPETS], props.userProfile)}
+                {makeToolbarDropdownItem(props.actions[ActionEnum.CODEOPTIONS], props.userProfile)}
```

Comment:
Having CodeSnippets and Code Options and AI transformations here is nonsensical. They pretty much have nothing in common. 

Maybe we can call this `Generated Code Options` or `Configure Code`

And also, we could move it to the top of this dropdown since it's the one that makes the most sense here and AI transofmriaton already has a toolbar button.